models.go

```diff
@@ -33,9 +33,13 @@ type Model struct {
 }
 
 type TextToSpeechRequest struct {
-	Text          string         `json:"text"`
-	ModelID       string         `json:"model_id,omitempty"`
-	VoiceSettings *VoiceSettings `json:"voice_settings,omitempty"`
+	Text               string         `json:"text"`
+	ModelID            string         `json:"model_id,omitempty"`
+	VoiceSettings      *VoiceSettings `json:"voice_settings,omitempty"`
+	PreviousRequestIds []string       `json:"previous_request_ids,omitempty"`
```

Comment:
see https://elevenlabs.io/docs/api-reference/text-to-speech/convert#request.body.previous_text